cms/djangoapps/contentstore/views/item.py

```diff
@@ -128,6 +132,9 @@ def xblock_handler(request, usage_key_string):
               these fields:
                 :parent_locator: parent for new xblock, required for both duplicate and create new instance
                 :duplicate_source_locator: if present, use this as the source for creating a duplicate copy
+                :move_source_locator: if present, use this as the source item for moving
+                :move_dest_locator: if present, use this as the destination parent for moving an item
```

Comment:
I would reuse ``parent_locator`` for this, rather than introducing a new field. It makes sense that "new" and "move" both specify the parent in the same way.